frontend/pages/dashboard/my-feed.tsx

```diff
@@ -1,26 +1,26 @@
 import { NextPage } from 'next'
 import { withApollo } from '../../lib/apollo'
-
+import { useTranslation } from '../../config/i18n'
 import DashboardLayout from '../../components/Layouts/DashboardLayout'
 import MyFeed from '../../components/Dashboard/MyFeed'
-
 import { useFeedQuery } from '../../generated/graphql'
 
 interface InitialProps {}
 
 interface Props extends InitialProps {}
```

Comment:
Shouldn't we add something to our Initial props interfaces on these pages now?

Reply:
Hmm, that's a good question... Right now `namespacesRequired` isn't ever used within the component, but I think there are use cases when you do need to reference one. I'll play around with it and test it out!

Reply:
Awesome sounds good! 😊

Reply:
I added `namespacesRequried: string[]` to `InitialProps`, but TypeScript doesn't like that Props inherits it without adding anything, so I just made it `<Props>`

Reply:
Cool, this feels better to me, thanks for looking into it 🙂